pom.xml

```diff
@@ -17,7 +17,7 @@
         <disableTestInjection>false</disableTestInjection>
         <java.level>8</java.level>
         <jackson.version>2.9.9</jackson.version>
-        <jenkins.version>2.162</jenkins.version>
+        <jenkins.version>2.184</jenkins.version>
```

Comment:
Okay, so:
The putty extension was removed as a dependency from Jenkins at some point (I couldn't see mention of it in the changelogs), which breaks the SSH credentials plugin. Explicitly adding it includes an explicit version update for the Trilead-ssh2 plugin and that requires Jenkins 2.184 as a minimum.
Hopefully I am mistaken, but from my understanding of maven, this means either we update our Jenkins minimum to that version, or we can't support SSH credentials above a certain version of Jenkins.

Reply:
Hmm, if we're bumping this we should go to 2.190.1 as that is the closest LTS. 
I'm happy for this, but if you do this you'll need to update our internal instance to the version you pick here.